src/lib/schematics/update/rules/attribute-selectors/attributeSelectorsStringLiteralRule.ts

```diff
@@ -0,0 +1,54 @@
+/**
```

Comment:
Probably in another PR, but we should also make these filenames consistent with the rest of the project (dashes instead of camelCase)